src/inference/include/ie/vpu/myriad_config.hpp

```diff
@@ -60,7 +60,7 @@ INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_VPU_CONFIG(MYRIAD_PCIE);
 INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_VPU_CONFIG(MYRIAD_USB);
 
 /**
- * @brief Optimize vpu plugin execution to maximize throughput.
+ * @brief Optimize npu plugin execution to maximize throughput.
```

Comment:
Often, when openvino code base mentions VPU, it actually means MYRIAD plugin. Is it supposed to be renamed to NPU as well?

Reply:
As far as I know the Myriad plugin was recently removed. It still remains only in the `releases/2022/3` branch.

Reply:
let's rename into `MYRIAD` in this place